Add documentation for using private image registries and a default workspace image for Self hosted (#2640)
gitpod-io/website: gitpod/docs/self-hosted/latest/advanced/default-workspace-image.md

---
section: self-hosted/latest
subsection: installation-guides
title: Private Registries
---

<script context="module">
  export const prerender = true;
</script>

> # Setting an installation-wide default workspace image

Reviewer: So, If they are using this, there is a good chance that they are also using private registries right for the workspace image? Should we link to the other doc from here? 🤔

Author: Fixed in the next commit :)

> This guide describes how to set a default workspace image that will apply to your entire Gitpod Self-Hosted installation. Please refer to the [docker image documentation](../../../config-docker) for more information about workspace images.

Reviewer 2: ```suggestion
This guide describes how to set a default workspace image that will apply to your entire Gitpod Self-Hosted installation. Please refer to the [Docker image documentation](../../../config-docker) for more information about workspace images.
```

> > **Note:** there are two ways of setting an image for a workspace: via the installation-wide default `workspaceImage` (mentioned here-) and the [`.gitpod.yml` file](../../../config-gitpod-file). The latter overwrites the former.

Reviewer 2: ```suggestion
> **Note:** there are two ways of setting an image for a workspace: via the installation-wide default `workspaceImage` (mentioned here) and the [`.gitpod.yml` file](../../../config-gitpod-file). The `.gitpod.yml` file will take precedence if both are set.
```

Fixes typo. Also, "the latter overwrites the former" might not be clear enough - hopefully, my suggestion is a good starting point
